cinder/volume/manager.py

```diff
@@ -680,6 +680,36 @@ def _set_resource_host(self, resource):
             resource.host = volume_utils.append_host(self.host, pool)
             resource.save()
 
+    def _propagate_volume_scheduler_hints(self, context, volume):
+        """Ensure metadata hints are propagated to referenced volumes."""
+        hints = volume_utils.get_scheduler_hints_from_volume(volume)
+        if not hints:
+            return
+
+        LOG.debug("Found hints for %(volume)s - %(hints)s",
+                  {'volume': volume.id, 'hints': hints})
+        for hint_key in hints:
+            for vol_id in hints[hint_key]:
+                try:
+                    meta_vol = objects.Volume.get_by_id(context, vol_id)
+                    # Because pep8 error otherwise with lines 696 and 710
```

Comment:
The lines don't match anymore. Maybe it would be best to not give explicit line-numbers and just refer to "further down"?

Reply:
done